packages/schema/src/config/adhoc.ts

```diff
@@ -29,6 +29,11 @@ export default defineUntypedSchema({
    * @type {typeof import('../src/types/imports').ImportsOptions}
    */
   imports: {
+    /**
+     * Toggle auto-imports for all composables, components, directories, and all other parts of your application that Nuxt usually handles.
+     */
+    autoImport: true,
```

Comment:
yes, good catch - this will mean it shows up in the `nuxt.config` generated markdown... You ca n preview the result by running `pnpm build:stub` (or `pnpm build`) and checking `packages/schema/schema/config.md` which will be generated.

It would be good to sync the docs between here and `../src/types/imports` so the text and examples are the same.

Reply:
Alright, thanks for confirming! Turns out preview was indeed the issue in displaying as it did not generate the page when going there, because at first I thought I broke it haha. Always glad to help!